Code/GraphMol/SLNParse/lex.yysln.cpp.cmake

```diff
@@ -17,10 +17,10 @@
 /* First, we deal with  platform-specific or compiler-specific issues. */
 
 /* begin standard C headers. */
-#include <stdio.h>
-#include <string.h>
-#include <errno.h>
-#include <stdlib.h>
+#include <cstdio>
```

Comment:
There's no need to update the PR, but modifying these machine-generated files doesn't make a lot of sense; the previous behavior will just be repeated the next time we update these files.